tools/dreport.d/ibm.d/package

```diff
@@ -4,8 +4,9 @@
 # and transferring to dump location.
 function custom_package()
 {
-    tar -Jcf "$name_dir.bin" -C \
-                 "$(dirname "$name_dir")" "$(basename "$name_dir")"
+    #using zstd compression
```

Comment:
the dump size is increasing but, I think this plugin is called only for the [p10](https://github.ibm.com/openbmc/openbmc/pull/2267/commits/0722b480c16201ed2e535f519768d282d2651a82#diff-b9dcea4197aa53ae9ebac1f8029a44885ac99a9bb695c85b8f0086771e766d9fR57) and [witherspoon tacoma](https://github.ibm.com/openbmc/openbmc/pull/2267/commits/0722b480c16201ed2e535f519768d282d2651a82#diff-b9dcea4197aa53ae9ebac1f8029a44885ac99a9bb695c85b8f0086771e766d9fR56) so I think we are good since we have more flash - is it correct?

Reply:
yes only p10